src/main/kotlin/org/oso/core/entities/HelpProvider.kt

```diff
@@ -1,22 +1,24 @@
 package org.oso.core.entities
 
 import com.fasterxml.jackson.annotation.JsonBackReference
-import javax.persistence.Column
-import javax.persistence.Entity
-import javax.persistence.ManyToMany
-import javax.persistence.Table
+import javax.persistence.*
+import javax.validation.constraints.Size
 
 @Entity
 @Table(name = "hp")
 data class HelpProvider(
-    @Column(unique = true)
+    @Id
+    @Column(nullable = false)
```

Comment:
Darf in der Datenbank nicht ``null`` sein aber in Kotlin schon?

Reply:
In DB the id cannot be null at any point, but it could be in code in case that the entity hasn't be stored yet. This way we can check if it is already in DB or not

Reply:
Ah I understand. 👍 

Another thing we should consider in the near future is disabling the generation of ddl code via jpa. The annotations would be redundant in that case.

Reply:
Indeed good point but i think this should not be part of this feature-branch

Reply:
I'll write an issue for it. 👍